README.md

```diff
@@ -300,7 +301,7 @@ Parameters:
 
 Returns: the result of the action
 
-### Examples
+#### Examples
```

Comment:
We can place the 'example' under the third-level heading in the final section.

Reply:
yes, please don't add these micro-sections. it makes the doc too nested.

Reply:
I indented this section because it looks like it had the wrong indentation to be an example of using `vscode.call`. I don't know where the 'final section' is that you want me to move this to so I'm just going to undo the change for now, but I can move it if you want.